product_specifications/34DbH5/igsn.feature

```diff
@@ -0,0 +1,89 @@
+
+Feature: Repository exclusive for IGSN-DOIs
+
+        Scenario: Creating a IGSN Catalogue in Fabrica
+            Given that a user is creating a repository in Fabrica
+             When the user chooses the repository type
+             Then the  repository type field should show the 'IGSN Catalogue' option
+             When the user selects 'IGSN Catalogue'
+             Then a info message should be displayed reading: "This repository will only be able to mind IGSN-DOIs."
+             When clicking create a new repository with type 'IGSN Catalogue' should be created.
+ 
+        Scenario: Editing a IGSN Catalogue in Fabrica
```

Comment:
Are we saying IGSN catalogues cannot be changed to just repository type?

Reply:
The main use case is about accurate reporting of IGSN. It has to do little with the users.  

Since, during the transition, DOI metadata would still not able to distinguish between DOIs and IGSN-DOIs, we are resourcing to artificial ways to manage IGSN-DOIs.

One of them is that once borne an IGSN-DOI they stay always like that (can never changer resourceType). But since  normal repositories do not have that restriction and they can modify metadata as they please allowing user to Transfer DOIs could lead to not only confusing DOIs but also shady reporting.